region/tourian/main/Metroid Room 2.json

```diff
@@ -343,6 +343,60 @@
         {"metroidFrames": 80}
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Very Deep Stuck X-Ray Climb With Ice",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "Ice",
+        "canXRayClimb"
+      ],
+      "bypassesDoorShell": true,
+      "note": [
+        "Enter with G-mode direct, and quickly freeze the Metroids (if still alive) and Rinkas.",
+        "Climb up 1 screen, moving quickly to avoid Rinka damage.",
+        "Perform a turnaround buffered spin-jump away from the door to trigger the transition, bypassing any lock on the door."
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Very Deep Stuck X-Ray Climb With Crystal Flash",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"or": [
+          "canArtificialMorph",
+          "f_KilledMetroidRoom2"
+        ]},
+        "h_canCrystalFlash",
+        {"autoReserveTrigger": {}},
+        "canXRayClimb",
+        {"enemyDamage": {
+          "enemy": "Rinka",
+          "type": "contact",
+          "hits": 1
```

Comment:
I think I took 3 hits, but I did kill all the rinkas before ending GMode.

Reply:
Yeah, what I found works best is to start climbing as soon as possible after the close one spawns, and lure it to spawn toward the left so it stays on screen as long as possible, buying time before its next spawn which will unavoidably hit. You can climb fast enough to make the far one miss you and to get safely far off-camera to prevent a second hit from the close one. The Rinkas don't move while X-ray is held, so it's fine to go slow while holding X-ray, just minimizing the time that X-ray is not held.